.devcontainer/welcome-message.txt

```diff
@@ -8,5 +8,6 @@
 ⭐⭐ =================================  IMPORTANT!!  ================================== ⭐⭐
    To complete setup of your development environment run the following in the terminal:
 
-      pip install -e .[all,test_all,docs]
+      pip install -U pip
+      pip install -e . --group test_all --group docs
```

Comment:
I think that is a net negative from a developer point of view - I use ``pip install .[...]`` all the time and now there is going to be more to type